src/pages/storage/forms/StoragePoolForm.tsx

```diff
@@ -28,6 +28,12 @@ export interface StoragePoolFormValues {
   ceph_rbd_clone_copy?: string;
   ceph_user_name?: string;
   ceph_rbd_features?: string;
+  powerflex_domain?: string;
+  powerflex_pool?: string;
+  powerflex_gateway?: string;
+  powerflex_user_password?: string;
```

Comment:
Thanks for this! I think we should also have `powerflex.user.name` configurable as this might be different than the default `admin` right from the beginning. I just saw https://github.com/canonical/lxd-ui/pull/676 which would be perfectly fine to tune or modify other settings afterwards.

Reply:
Nice, thanks!